jwst/source_catalog/source_catalog.py

```diff
@@ -832,83 +1020,108 @@ def nn_dist(self):
     @lazyproperty
     def aper_total_flux(self):
         """
-        The aperture-corrected total flux for sources, based on the flux
-        in largest aperture.
+        Return the aperture-corrected total flux for sources.
 
-        The aperture-corrected total flux should be used only for
-        unresolved sources.
+        Computed based on the flux in largest aperture.Should be used only for unresolved sources.
+
+        Returns
+        -------
+        `~astropy.unit.Quantity`
+            The aperture-corrected total flux
```

Comment:
Is it useful to indicate that this is not single valued, i.e. an array of floats with units attached?  Not sure the best way to indicate this.

Reply:
Good point, actually before looking into this I was not aware that Quantity objects can be array-like.  I will add something to that effect in the docstring

Reply:
How does https://github.com/spacetelescope/jwst/pull/9138/commits/bf2e842a361ecd7aae84b40eafaa1df72157657d look?